lib/har/page_timings.rb

```diff
@@ -2,17 +2,15 @@ module HAR
   class PageTimings < SchemaType
     def initialize(input)
       super(input)
-      define_custom_timings(input)
+      @input = input
     end
 
-    private
+    def custom
+      @custom ||= begin
+        custom_timings = @input.select { |key| key.start_with?('_') }
+                               .transform_keys { |key| key[1..-1] }
```

Comment:
Looks like `transform_keys` is breaking the build for some older Ruby versions so we should fix that.

Reply:
Fixed. Replaced it with map.